Controller/AuthorizeController.php

```diff
@@ -52,14 +52,14 @@ public function authorizeAction(Request $request)
         $form = $this->container->get('fos_oauth_server.authorize.form');
         $formHandler = $this->container->get('fos_oauth_server.authorize.form.handler');
 
+        $scope = $this->container->get('request')->get('scope', null);
```

Comment:
This is incomplete. When the form is submitted, `$scope` will be null instead of the scope submitted by the form.
This may pose problems to consumers of `OAuthEvent`, because they suddenly receive an empty scope when the form is submitted. If this influences the resut of `$event->isAuthorizedClient()` so that it becomes `true` while it should have returned `false` for these scopes, this is a security issue. (I tried to fix this #332, where I reimplemented this PR because i did not look down the PR list far enough)